.github/workflows/test-gb-25.yml

```diff
@@ -53,8 +53,8 @@ jobs:
           - 'main'
           # - '0123456789abcdef0123456789abcdef01234567'
         reactant_commit:
-          - 'main'
-          # - '6bb1f30309ba401d4fdd654dafb48c08c73503f7'
+          # - 'main'
+          - "bde0d3b2349cbaa2790b3edd5da21be55b943586"
```

Comment:
Please change this back to `main` after https://github.com/EnzymeAD/Reactant.jl/pull/1538 is merged.